test/arrays/userAPI/assocArrayAPItest.chpl

```diff
@@ -16,7 +19,18 @@ proc readArray(ref X) {
   X["ten"]   = 10.0;
 }
 
-proc testAssocArrayAPI(ref X: []) {
+proc testAssocArrayAPI(ref X: [], output:fileWriter(?)=stdout) {
```

Comment:
Could you rename `output` to `arrayOutput` ? I think that would make it a bit clearer what is going on.